### DISCUSSION TITLE
不会就只有我有Github Pro和网站吧

### DISCUSSION THREAD

**Discussion by fxlqwq** [share121/inter-knot#2042] [其他]
# 不会就只有我有Github Pro和网站吧
![image](https://github.com/user-attachments/assets/44cc8c6c-0c5c-40c1-84c7-0972b3892e29)
![image](https://github.com/user-attachments/assets/44cc8c6c-0c5c-40c1-84c7-0972b3892e29)

**Comment by MoYuan-CN** [comment]
![image](https://github.com/user-attachments/assets/29a77647-4d6f-41c1-8f7a-bf20fd190d33)
你 Achievements 一个都没有么……
如果是这样，我有足够的理由证明你是来炫富而不是展示技能的……
